app/controllers/ratings_controller.rb

```diff
@@ -0,0 +1,19 @@
+class RatingsController < ApplicationController
+
+  def create
+    rating = current_student.ratings.new(rating_params)
+    if rating.save
+    else
+      rating = Rating.where(internship_id: params[:internship_id], student: current_student).first
+      rating.update(rating_params)
+    end
```

Comment:
refactor lines 4-7 to use `Rating.for(current_student, internship)` (as I described above)